ffi/SDL2_0.lua

```diff
@@ -39,6 +39,14 @@ local function SDL_VersionNum(x, y, z)
     return x*1000 + y*100 + z
 end
 
+-- Quick and dirty way of checking if we're running on a ChromeOS container.
+-- see https://chromium.googlesource.com/chromiumos/docs/+/master/containers_and_vms.md#overview
+local function SDL_Runs_On_ChromeOS()
+    if jit.os ~= "Linux" then return false end
+    -- the sommelier binary is bind-mounted in every container
+    return os.execute("which sommelier >/dev/null 2>&1") == 0
```

Comment:
Not sure if this is called once or may be called a lot of times - but if the latter, the result could be cached (or the function replaced in-place by one that returns true or false).